Core/clim-core/frames.lisp

```diff
@@ -1668,13 +1670,14 @@ have a `pointer-documentation-pane' as pointer documentation,
 	      (when last-presentation
 		(do-hilite last-presentation (last-window) :unhighlight))
 	      (setq last-presentation presentation
-		    feedback-fn (feedback dest-translator)
-		    hilite-fn (highlighting dest-translator))
+		    feedback-fn       (when dest-translator (feedback dest-translator))
+		    hilite-fn         (when dest-translator (highlighting dest-translator)))
 	      (do-hilite presentation window :highlight)
 	      (do-feedback window x y :highlight)
-	      (document-drag-n-drop dest-translator presentation
-				    context-type frame event window
-				    x y)))
+	      (when dest-translator
```

Comment:
imho this should be handled in document-drag-n-drop, something in a spirit of diff below. rationale: this is a local function to frame-drag-and-drop (I guess it was put standalone due to length), such handling would be consistent with what you did with `do-highlight` and `do-feedback`.

```diff
 (defun document-drag-n-drop
     (translator presentation context-type frame event window x y)
-  (when *pointer-documentation-output*
-    (let ((s *pointer-documentation-output*))
-      (window-clear s)
-      (with-end-of-page-action (s :allow)
-	(with-end-of-line-action (s :allow)
-	  (funcall (pointer-documentation translator)
-	   *dragged-object*
-	   :presentation *dragged-presentation*
-	   :destination-object (and presentation
-				    (presentation-object presentation))
-	   :destination-presentation presentation
-	   :context-type context-type
-	   :frame frame
-	   :event event
-	   :window window
-	   :x x
-	   :y y
-	   :stream s))))))
+  (alexandria:when-let* ((s *pointer-documentation-output*)
+                         (pd (and translator (pointer-documentation translator))))
+    (window-clear s)
+    (with-end-of-page-action (s :allow)
+      (with-end-of-line-action (s :allow)
+        (funcall pd *dragged-object*
+                 :presentation *dragged-presentation*
+                 :destination-object (and presentation
+                                          (presentation-object presentation))
+                 :destination-presentation presentation
+                 :context-type context-type
+                 :frame frame
+                 :event event
+                 :window window
+                 :x x
+                 :y y
+                 :stream s)))))
```

Reply:
Good idea. I'm changing the commit accordingly.